subsys/dfu/dfu_target/Kconfig

```diff
@@ -117,14 +117,37 @@ config DFU_TARGET_SUIT
 	bool "SUIT DFU target support"
 	default y
 	depends on SUIT
+	depends on SDFW_SERVICES_ENABLED
 	imply MPU_ALLOW_FLASH_WRITE
 	select SUIT_UTILS
 	select ZCBOR
 	select ZCBOR_CANONICAL
-	select DFU_TARGET_STREAM
+	imply FLASH
+	imply STREAM_FLASH
```

Comment:
Shouldn't we replace imply with select if the dfu target suit implementation unconditionally requires using stream flash?